src/javascript/app_2/Modules/Trading/Containers/SettingsDialog/settings_chart.jsx

```diff
@@ -0,0 +1,51 @@
+import React           from 'react';
+import PropTypes       from 'prop-types';
+import SettingsControl from '../../../../App/Components/Elements/SettingsDialog/settings_control.jsx';
+import { connect }     from '../../../../Stores/connect';
+
+class ChartSettings extends React.Component {
+    render() {
+        return (
+            <div className='tab-content'>
+                <div className='chart-setting-container'>
+                    <SettingsControl
+                        name='position'
+                        toggle={this.props.toggleLayout}
+                        to_toggle={this.props.is_layout_default}
+                        style='toggle-chart-layout'
+                    />
+                    <SettingsControl
+                        name='asset information'
+                        toggle={this.props.toggleAsset}
+                        to_toggle={this.props.is_asset_visible}
+                    />
+                    <SettingsControl
+                        name='scale countdown'
+                        toggle={this.props.toggleCountdown}
+                        to_toggle={this.props.is_countdown_visible}
+                    />
+                </div>
+            </div>
+        );
+    }
+};
+
```

Comment:
this component class is stateless, perhaps better to use functional? `const ChartSettings = () =>(...)`

Reply:
Yep, working on it. :)